ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -695,7 +704,11 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
         hints_dup->caps &= ~(FI_LOCAL_COMM | FI_REMOTE_COMM);
         hints_dup->fabric_attr->prov_name = strdup("efa");
 
-        ret = fi_getinfo(fi_version, NULL, NULL, 0ULL, hints_dup, &providers);
+        ret = fi_getinfo(fi_primary_version, NULL, NULL, 0ULL, hints_dup, &providers);
+        if (FI_ENOSYS == -ret) {
```

Comment:
I couldn't find in the man page that FI_ENOSYS is returned when version doesn't match, if this is the case, is it possible to add it to the man page?

Reply:
Sure. I am engaging with the community to try to clarify

Reply:
Libfabric document updated:

https://ofiwg.github.io/libfabric/main/man/fi_getinfo.3.html